src/index.test.js

```diff
@@ -79,6 +73,81 @@ describe('index', () => {
     expect(actualTags).toEqual([{ key: 'k0', value: 'v0' }, { key: 'k1', value: 'v1' }]);
   });
 
+  test('logs - error (should filter long entries and cut after the 10s element)', async () => {
+    const consoleLog = jest.spyOn(console, 'log');
```

Comment:
you can do it in the initialization of the the `describe`, aside the other spies